src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md

```diff
@@ -14,6 +14,10 @@
   occurs.
   ([#3407](https://github.com/open-telemetry/opentelemetry-dotnet/issues/3407))
 
+* Create spans for HTTP retries and redirects according to
```

Comment:
could you describe the change here itself for users benefit? 
Having to read spec and PR to understand what changed is too complicated for end users.